timesketch/lib/analyzers/expert_sessionizers_test.py

```diff
@@ -22,30 +23,28 @@ class TestWebActivitySessionizerPlugin(BaseTest, BaseSessionizerTest):
     def test_event_type(self):
         """Test the mocking of events returns an event stream that matches the
         query for the analyzer."""
-        with mock.patch.object(
-                self.analyzer_class,
-                'event_stream',
-                return_value=_create_mock_event(
-                    0,
-                    2,
-                    source_attrs={'source_short': 'WEBHIST'})):
-            index = 'test_index'
-            sketch_id = 1
-            analyzer = self.analyzer_class(index, sketch_id)
-            message = analyzer.run()
-            self.assertEqual(
-                message,
-                'Sessionizing completed, number of session created: 1')
-
-            ds = MockDataStore('test', 0)
-            event1 = (ds.get_event('test_index', '0', stored_events=True))
-            self.assertEqual(event1['_source']['source_short'], 'WEBHIST')
-            self.assertEqual(event1['_source']['session_id'],
-                             {analyzer.session_type: 1})
-            event2 = (ds.get_event('test_index', '101', stored_events=True))
-            self.assertEqual(event2['_source']['source_short'], 'WEBHIST')
-            self.assertEqual(event2['_source']['session_id'],
-                             {analyzer.session_type: 1})
+        index = 'test_index'
+        sketch_id = 1
+        analyzer = self.analyzer_class(index, sketch_id)
+        analyzer.datastore.client = mock.Mock()
+        ds = analyzer.datastore
+
+        _create_mock_event(ds, 0, 2, source_attrs={'source_short': 'WEBHIST'})
+
+        message = analyzer.run()
+        self.assertEqual(
+            message, 'Sessionizing completed, number of session created: 1')
+
+        # pylint: disable=unexpected-keyword-arg
```

Comment:
get_event() in MockDataStore is getting one argument more that ElasticsearchDataStore, and since ds have to be instance of ElasticsearchDataStore if not mocked, pylint is throwing and error "Unexpected keyword argument 'stored_events' in method call"